acceptance/phase_test.go

```diff
@@ -144,7 +145,7 @@ func (d *targetDaemon) createFixtures(t *testing.T) {
 	var fixtures daemonImageFixtures
 
 	appMeta := minifyMetadata(t, filepath.Join("testdata", "app_image_metadata.json"), platform.LayersMetadata{})
-	cacheMeta := minifyMetadata(t, filepath.Join("testdata", "cache_image_metadata.json"), platform.CacheMetadata{})
+	cacheMeta := minifyMetadata(t, filepath.Join("testdata", "cache_image_metadata.json"), cache.Metadata{})
```

Comment:
I originally moved this struct to try and avoid an import cycle. I think now it could move back to the platform package, but maybe it's better here (image and volume caches have `SetMetadata(metadata Metadata)` and `RetrieveMetadata()`).

Reply:
Moved it back to make the diff smaller...